### PR TITLE
Add new WYSIWYG Full interface

### PR DESCRIPTION
**Pull request by rijkvanzanten**
It's a more full-fledged alternative to the Medium-style WYSIWYG editor. It's based on TinyMCE.

pasted_image_at_2017_05_26_12_25_am_720
pasted_image_at_2017_05_26_12_26_am_360-1
pasted_image_at_2017_05_26_12_26_am_360

### CODE REVIEW

File: app/utils.js [hunk 1]
```diff
@@ -129,6 +129,73 @@ define(['underscore'], function (_) {
     return value == null || value === '';
   };
 
+  /**
+   * Returns a function, that, as long as it continues to be invoked, will not be triggerd.
+   * The function will be called after it stops being called for N ms.
+   * Based on https://github.com/component/debounce/blob/master/index.js which in turn is based on the
+   * original implemenation in Underscore (both licensed MIT)
+   */
+  Utils.debounce = function(func, wait, immediate) {
```
Comment (WellingGuzman) [inline app/utils.js]: Are we still going to use this debounce? We could use the underscore implementation

Comment (rijkvanzanten) [inline app/utils.js]: It's used now, but the implementation and functionality match that of the underscore one. Underscore isn't used in hardly any of the interfaces, so we could discuss keeping the interfaces underscore-free to keep it a little more modular and easier to move away from underscore and save 20kb?

Comment (rijkvanzanten) [inline app/utils.js]: Whatever you think is best. It doesn't really matter on the short term

Comment (WellingGuzman) [inline app/utils.js]: We use a lot of underscore utils over the place and it's already loaded so it won't make much pain.

We avoid to have to functions doing the same thing, and underscore will be proxy/wrapped by Utils some day.

You can add a function inside the interface so when it's time to move it, if, then add the code in there and should be good to go.

Or, use the underscore one inside the Utils, and that should be good to go too.

Comment (rijkvanzanten) [inline app/utils.js]: That would result in something like `Utils.debounce = _.debounce`, which also seams like an abstraction which would do more harm than good. I'll replace it with the underscore one for now 👍

Comment (WellingGuzman) [inline app/utils.js]: what harm would that do?

Comment (rijkvanzanten) [inline app/utils.js]: Ah, you meant to be able to replace it at a later time. Yeah that's no problem ^^ (little brainfart here)

**Pull request merged**